man/systemd.service.xml

```diff
@@ -513,6 +513,12 @@
         <varname>Type=oneshot</varname> is used, in which case the
         timeout is disabled by default (see
         <citerefentry><refentrytitle>systemd-system.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>).
+        </para>
+
+        <para><varname>Type=notify</varname> services may by sending <varname>EXTEND_TIMEOUT=1</varname> cause
+        this value to be ignored after the first receipt of this message and will thereafter rely on less that
+        watchdog interval repeats of this message until the service is started (see
+        <citerefentry><refentrytitle>sd_notify</refentrytitle><manvolnum>3</manvolnum></citerefentry>).
```

Comment:
This *is* grammatically correct, but somehow hard to parse. Maybe
> If a service of Type=notify sends an EXTEND_TIMEOUT=1 notification, which it may do more than once, the deadline for startup is replaced. The new deadline is watchdog interval (see WatchdogIntervalSec=) after the reception of that notification.